src/libsystemd-network/sd-dhcp-lease.c

```diff
@@ -32,6 +32,11 @@
 #include "tmpfile-util.h"
 #include "unaligned.h"
 
+/* rfc8925 - IPv6-Only Preferred Option for DHCPv4 3.4.
+ * MIN_V6ONLY_WAIT: The lower boundary for V6ONLY_WAIT. Value: 300 seconds
+ */
```

Comment:
Not a big matter, but please move `*/` at the end of the previous line. Please make the comment shorter.